index.html

```diff
@@ -0,0 +1,155 @@
+    </div>
+
+</header>
+<main class="outer-container">
+    <section>
+        <div class="inner-container-narrow">
+            <p class="about-us">
+                Rekupe is a digital agency in Los Angeles.
+                <br>
+                <br>
+                We use strategic UX, SEO optimization and beautiful visual designs to help your business grow.
+                <br>
+                <br>
+                We see every project as a fresh opportunity, pushing ourselves to create new digital experiences.
+            </p>
+
+            <p class="contact-us">
+                Have a question? <a href="#contact-us">Click here</a> to contact us!
+            </p>
+        </div>
+
+    </section>
+
+    <section class="cases">
+        <div class="inner-container-wide">
+            <article class="case">
+                <div class="cases-text">
+                    <h5>Rolling Stone</h5>
+                    <img src="assets/squiggle_line.svg" alt="squiggle line" width="20px">
+                    <p>You know Rolling Stone. It’s the definitive voice in music, politics and culture. It’s a global
+                        icon.</p>
+                    <H6>VIEW PROJECT</H6>
+                </div>
+                <span class="spacer">
+                </span>
+                <span class="image-wrapper">
+                    <img src="assets/work-image-01.jpeg" alt="rolling stones"/>
+                </span>
+            </article>
+
+            <article class="case">
+                <div class="cases-text">
+                    <h5>Variety</h5>
+                    <img src="assets/squiggle_line.svg" alt="squiggle line" width="20px">
+                    <p>Variety is an industry leader for breaking entertainment news, reviews and box office results
+                        with archive of content dating back to 1905.</p>
+                    <H6>VIEW PROJECT</H6>
+                </div>
+                <span class="spacer">
+                </span>
+                <span class="image-wrapper">
+                    <img src="assets/work-image-02.jpeg" alt="variety image"/>
+                </span>
+            </article>
+
+            <article class="case">
+                <div class="cases-text">
+                    <h5>Worth</h5>
+                    <img src="assets/squiggle_line.svg" alt="squiggle line" width="20px">
+                    <p>A global media platform connecting a community that embraces worth beyond wealth.</p>
+                    <H6>VIEW PROJECT</H6>
+                </div>
+                <span class="spacer">
+                </span>
+                <span class="image-wrapper">
+                    <img src="assets/work-image-03.jpeg" alt="worth image"/>
+                </span>
+            </article>
+        </div>
+    </section>
+
+    <section>
+        <div class="services inner-container-wide">
+            <article class="service-block">
```

Comment:
De kaartjes staan onder elkaar in plaats van 2 rijen van 3 kaartjes zoals in het voorbeeld
![Scherm­afbeelding 2023-01-21 om 12 21 57](https://user-images.githubusercontent.com/23332341/213864729-a44e60c2-bcfc-4e43-bf53-4fd9537d470c.jpg)